machine-learning/sagemaker-edge-manager/artifacts/com.greengrass.SageMakerEdgeManager.ObjectDetection/1.0.0/object_detection/sagemaker_edge_agent_pb2.py

```diff
@@ -0,0 +1,83 @@
+
+_builder.BuildMessageAndEnumDescriptors(DESCRIPTOR, globals())
+_builder.BuildTopDescriptorsAndMessages(DESCRIPTOR, 'sagemaker_edge_agent_pb2', globals())
+if _descriptor._USE_C_DESCRIPTORS == False:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The `==` and `!=` operators use the compared objects&#39; ` __eq__` method to test if they are equal. To check if an object is a singleton, such as `None`, we recommend that you use the `is` identity comparison operator.

[Learn more](https://docs.python.org/3/reference/expressions.html#is)